### PR TITLE
Eqf

### PR DESCRIPTION
**Pull request by richared**
Added the possibility for eqf level and other qualification framework levels.

### CODE REVIEW

**Review by mpuzar — APPROVED** [review]

**Comment by wrygiel** [comment]
Which base issue is this PR about? I cannot find anything with "qualification" in our [TODO list](https://github.com/emrex-eu/elmo-schemas/issues).

**Comment by richared** [comment]
This PR is based on needs that have arisen during work with new countries that wish to join EMREX, in order for them to join the network, they need this change, as I understand it. This is also to support more than just the basic higher education.

**Comment by wrygiel** [comment]
But have these changes been discussed in Oslo, or is it something completely new?

**Comment by richared** [comment]
It is connected to issue 26. I just missed when I wrote the branch name... Sorry.

File: schema.xsd [hunk 1]
```diff
@@ -1048,6 +1048,7 @@
                                                         </xs:documentation>
                                                     </xs:annotation>
                                                 </xs:element>
+                                                <!-- DEPRICATED - will be removed in Version 2-->
```
Comment (mpuzar) [inline schema.xsd]: "deprEcated".

**Pull request merged**